examples/scripts/compare_comsol/compare_comsol_DFN.py

```diff
@@ -140,9 +140,11 @@ def myinterp(t):
 
 # Make new solution with same t and y
 comsol_solution = pybamm.Solution(pybamm_solution.t, pybamm_solution.y)
-# Update model scales to match the pybamm model
+# Update model and solution scales to match the pybamm model
 comsol_model.timescale = pybamm_model.timescale
 comsol_model.length_scales = pybamm_model.length_scales
```

Comment:
you probably don't need these two lines any more now that the solution contains the timescale

Reply:
I've removed the unnecessary lines now